src/transform3d.rs

```diff
@@ -177,6 +177,18 @@ where T: Copy + Clone +
         )
     }
 
+    /// Check whether shapes on the XY plane with Z pointing towards the
+    /// screen transformed by this matrix would be facing back.
```

Comment:
It implies "something front-facing being transformed will become back-facing" which is maybe a bit convoluted but I believe it conveys the correct information.